assets/images/web-stories/block/carousel-arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="14" viewBox="0 0 9 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
nitpic, can we put this file in an icons directory, to make it is a little cleaner.

Reply:
you mean to create a new directory `icons` and keep this svg there right?

Reply:
moved svg to `assets/src/web-stories-block/icons/`.